### ISSUE TITLE
Multi-checkbox code bug

### ISSUE THREAD

**Issue by kentcdodds** [formly-js/angular-formly-templates-bootstrap#35]
[This line](https://github.com/formly-js/angular-formly-templates-bootstrap/blob/master/src/types/multiCheckbox.js#L74) reads:

```javascript
if(opts.expressionProperties && opts.expressionProperties.required){
```

This will never evaluate to `true` because `required` is an invalid property for the root of the form config. It should read:

```javascript
if(opts.expressionProperties && opts.expressionProperties['templateOptions.required']){
```

Sanity check? @ckniffen

**Comment by ckniffen** [comment]
Very sane. Great catch.

**Comment by kentcdodds** [comment]
Released as `4.4.1`
